docs/assemblies/proc_creating-a-set-of-data-plane-nodes.adoc

```diff
@@ -8,10 +8,21 @@ Create an `OpenStackDataPlaneNodeSet` CR for each logical grouping of nodes in y
 
 .Procedure
 
-. Copy the sample `OpenStackDataPlaneNodeSet` CR that you require and save it to a file named `openstack-edpm.yaml` on your workstation:
+. Copy the https://kustomize.io/[kustomize] examples on your workstation:
 
-* Pre-provisioned nodes: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml
-* Bare metal nodes: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml
+  git clone -n --depth=1 --filter=tree:0 \
+    https://github.com/openstack-k8s-operators/dataplane-operator
+  cd dataplane-operator
+  git sparse-checkout set --no-cone examples
+  git checkout
+
+  * Pre-provisioned nodes:
```

Comment:
I think you should add text here like:

"Use kustomize to create the `OpenStackDataPlaneDeployment` and `OpenStackDataPlaneNodeSet` CRs."

You could also add:

"If desired, update the values.yaml file with values from your environment and add other kustomizations as needed before generating the CRs. Alternatively, the generated CRS can be directly edited before being applied."